markdown/ch3.markdown

```diff
@@ -848,8 +848,14 @@ from scikit-image — indeed, the library that contains this chapter's code snip
 
 ```python
 from skimage.future import graph
+
 g = graph.rag_mean_color(tiger, seg)
-graph.show_rag(seg, g, tiger);
+
+# Remove after release of skimage v0.13
+import skimage
+if skimage.__version__ >= '0.13':
+
+    graph.show_rag(seg, g, tiger);
```

Comment:
I wasn't sure whether the function only got renamed, or whether the API changed too, so this was the Quick Fix.

Let's see:

```diff
-def draw_rag(labels, rag, img, border_color=None, node_color='#ffff00',
-             edge_color='#00ff00', colormap=None, thresh=np.inf,
-             desaturate=False, in_place=True):
+def show_rag(labels, rag, img, border_color='black', edge_width=1.5,
+             edge_cmap='magma', img_cmap='bone', in_place=True, ax=None):
           edge_cmap='magma', img_cmap='bone', in_place=True, ax=None):
```

Yup, that should work, will change.